astropy/table/np_utils.py

```diff
@@ -107,7 +103,7 @@ def get_descrs(arrays, col_name_map):
         # Make sure all input shapes are the same
         uniq_shapes = set(col.shape[1:] for col in in_cols)
         if len(uniq_shapes) != 1:
-            raise TableMergeError(f'Key columns {name!r} have different shape')
+            raise TableMergeError(f'Key columns have different shape')
```

Comment:
This is a real bug that showed up where an undefined variable was used in an exception.  Bad coverage...